src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -197,16 +200,11 @@ class SidebarLinks extends React.Component {
                             ]}
                             data={optionListItems}
                             focusedIndex={_.findIndex(optionListItems, (
-                                option => option.toString() === this.props.reportIDFromRoute
+                                option => option.toString() === this.props.currentReportId
                             ))}
                             onSelectRow={this.showReportPage}
                             shouldDisableFocusOptions={this.props.isSmallScreenWidth}
                             optionMode={this.props.priorityMode === CONST.PRIORITY_MODE.GSD ? CONST.OPTION_MODE.COMPACT : CONST.OPTION_MODE.DEFAULT}
-                            onLayout={() => {
-                                this.props.onLayout();
-                                App.setSidebarLoaded();
-                                this.isSidebarLoaded = true;
-                            }}
```

Comment:
I think there was a problem with `onLayout` prop on some platform
cc @staszekscp

Reply:
It was not firing on mWeb Safari at all, so it had to be moved to `componentDidMount` instead to keep the correct flow. Especially because we did not use event that is passed to the `onLayout` function